src/main/java/cz/cvut/kbss/study/security/SecurityConstants.java

```diff
@@ -27,7 +27,32 @@ private SecurityConstants() {
      */
     public static final int SESSION_TIMEOUT = 12 * 60 * 60;
 
-    public static final String ROLE_USER = "ROLE_USER";
+    public static final String user = "user";
```

Comment:
Why not using Role enum? Why we need to duplicate?

Reply:
Some methods, such as hasRole or hasAuthority, require constants. However, with enums, you can only retrieve the name by calling the name() method. That is why I made these constants for using values.